frontend/DashboardPage.tsx

```diff
@@ -0,0 +1,177 @@
+    },
+
+    barName: {
+        fontSize: '1rem',
+        fontWeight: 'bold',
+        cursor: 'pointer',
+    },
+    mainbar: {
+        cursor: 'pointer',
+        padding: 10,
+    },
+    main:{
+        fontSize: '1rem',
+        fontWeight: 'bold',
+        cursor: 'pointer',
+    },
+    mailSection:{
+        flexGrow: 1,
+    }
+
+   
+
+});
+
+
+export default function DashboardPage() {
+    const classes = useStyles();
+
+    // Add the options for the sidebar to the Dashboard list.
+    var options: DashboardItems[] = [
+        {
+            icon: <MailIcon />,
+            optionsName: 'Penpal Mails',
+        },
+        {
+            icon: <VideoCallIcon />,
+            optionsName: 'Start Video Call',
+        },
+        {
+            icon: <SearchIcon />,
+            optionsName: 'Find Penpal',
+        },
+        {
+            icon: <EditIcon />,
+            optionsName: 'Edit Profile',
+        },
+        {
+            icon: <InfoIcon />,
+            optionsName: 'Tutorials',
+        },
+    ];
+
+    var contents: Maillist[] = [
+        {
+            profile: <AccountCircleIcon />,
+            name: 'John Doe',
+            message: 'New Video',
+            dateReceived: 'Oct 22',
+        },
+        {
+            profile: <AccountCircleIcon />,
+            name: 'John Doe Jr.',
+            message: 'New Video',
+            dateReceived: 'Oct 21',
+        },
+        {
+            profile: <AccountCircleIcon />,
+            name: 'Thomas Jones',
+            message: 'New Video',
+            dateReceived: 'Oct 20',
+        },
+        {
+            profile: <AccountCircleIcon />,
+            name: 'John Doe Jr',
+            message: 'New Video',
+            dateReceived: 'Oct 19',
+        },
+        {
+            profile: <AccountCircleIcon />,
+            name: 'John Doe',
+            message: 'New Video',
+            dateReceived: 'Oct 18',
+        },
+        
+    ];
+
+
+    return (
+        
+    <>
+        <div style={{ margin: 0, padding: 0, boxSizing: 'border-box'}}>
+            <nav style={{ display: "flex", justifyContent: 'space-around', minHeight: '10vh', alignItems:'center', backgroundColor: '#CE0A2E', color: 'white'}}>
+                <div className="greeting" style={{ fontSize: '18px', fontWeight: 'normal' }}>Hi, Jane</div>
+                <ul className="nav-pages" style={{ listStyleType: 'none', display: 'flex', justifyContent: 'space-around', width: "30%", fontSize: '18px', fontWeight:'normal'}}>
+                    <LinkItem to='/' text='Home'/>
+                    <LinkItem to='/About' text='About' />
+                    <LinkItem to='/ContactUs' text='Contact Us' />
+                    <LinkItem to='/Help' text='Help' />
+                    <LinkItem to='/LogOut' text='Log Out' />
+                    
+                </ul>
+            </nav>
+        </div>
+
+        <div className={classes.mailSection}>
+        <Grid container>

```

Comment:
It looks like your spacing here is a bit off. Also it doesn't make sense to wrap a grid in a div. Grid has a `className` prop, so you can apply the css to it.